fastcrypto/src/groups/multiplier/windowed.rs

```diff
@@ -45,24 +50,29 @@ impl<
 }
 
 impl<
-        G: GroupElement<ScalarType = S>,
-        S: GroupElement + ToFromByteArray<SCALAR_SIZE>,
+        G: for<'a> Add<&'a G, Output = G> + for<'a> Mul<&'a S, Output = G> + Doubling + Clone + Debug,
```

Comment:
They already are part of GroupElement. I've done it like this because the goal of the PR is to avoid using GroupElement as the trait bound to also allow ParameterizedGroupElement. To do this, I've restricted the trait bounds as much as I can, namely to the operators actually used.